superagent/superagent.d.ts

```diff
@@ -77,6 +77,7 @@ declare module "superagent" {
       notAcceptable: boolean;
       notFound: boolean;
       forbidden: boolean;
+      xhr: XMLHttpRequest;
```

Comment:
I get a TS2304 error `Cannot find name 'XMLHttpRequest'` because of line 80 `xhr: XMLHttpRequest;`